[2024-08-12T09:39:08.199+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-12T09:39:08.216+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_prueba_tecnica.ingesta manual__2024-08-12T09:39:03.023364+00:00 [queued]>
[2024-08-12T09:39:08.221+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_prueba_tecnica.ingesta manual__2024-08-12T09:39:03.023364+00:00 [queued]>
[2024-08-12T09:39:08.221+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-08-12T09:39:08.231+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): ingesta> on 2024-08-12 09:39:03.023364+00:00
[2024-08-12T09:39:08.237+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=160) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-12T09:39:08.238+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_prueba_tecnica', 'ingesta', 'manual__2024-08-12T09:39:03.023364+00:00', '--job-id', '235', '--raw', '--subdir', 'DAGS_FOLDER/dag_prueba_tecnica.py', '--cfg-path', '/tmp/tmplrxduf9h']
[2024-08-12T09:39:08.239+0000] {standard_task_runner.py:64} INFO - Started process 173 to run task
[2024-08-12T09:39:08.239+0000] {standard_task_runner.py:91} INFO - Job 235: Subtask ingesta
[2024-08-12T09:39:08.269+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_prueba_tecnica.ingesta manual__2024-08-12T09:39:03.023364+00:00 [running]> on host 0ad27a48216c
[2024-08-12T09:39:08.327+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='miguel_guimarey' AIRFLOW_CTX_DAG_ID='dag_prueba_tecnica' AIRFLOW_CTX_TASK_ID='ingesta' AIRFLOW_CTX_EXECUTION_DATE='2024-08-12T09:39:03.023364+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-12T09:39:03.023364+00:00'
[2024-08-12T09:39:08.328+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-12T09:39:09.245+0000] {warnings.py:112} WARNING - /opt/***/dags/dag_prueba_tecnica.py:32: DtypeWarning: Columns (46,56,59) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv('/opt/***/dags/dataset.csv', delimiter=';').to_json(orient='records')

[2024-08-12T09:39:10.570+0000] {logging_mixin.py:188} INFO - Se lee el dataset de datos
[2024-08-12T09:39:17.127+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-12T09:39:17.128+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-12T09:39:17.135+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_prueba_tecnica, task_id=ingesta, run_id=manual__2024-08-12T09:39:03.023364+00:00, execution_date=20240812T093903, start_date=20240812T093908, end_date=20240812T093917
[2024-08-12T09:39:17.181+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-12T09:39:17.200+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-12T09:39:17.204+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
